src/OpenLoco/GameCommands/LoadSaveQuit.cpp

```diff
@@ -0,0 +1,258 @@
+        }
+        else if (!isNetworked() && loadSaveGameOpen())
+        {
+            // 0x0043C033
+            auto path = fs::path(&_savePath[0]).replace_extension(S5::extensionSV5).u8string();
+            std::strncpy(&_currentScenarioFilename[0], path.c_str(), std::size(_currentScenarioFilename));
+
+            if (sub_441FA7())
+            {
+                resetScreenAge();
+                throw GameException::Interrupt;
+            }
+        }
+        else if (isNetworked())
+        {
+            // 0x0043C0DB
+            if (CompanyManager::getControllingId() == CompanyManager::updatingCompanyId())
+            {
+                MultiPlayer::setFlag(MultiPlayer::flags::flag_4);
+                MultiPlayer::setFlag(MultiPlayer::flags::flag_3);
+            }
+        }
+
+        // 0x0043C0D1
+        Gfx::invalidateScreen();
+        return 0;
+    }
+
+    // 0x0043C182
+    static uint32_t quitGame()
+    {
+        _game_command_nest_level = 0;
+
+        // Path for networked games; untested.
+        if (isNetworked())
+        {
+            clearScreenFlag(ScreenFlags::networked);
+            auto playerCompanyId = CompanyManager::getControllingId();
+            auto previousUpdatingId = CompanyManager::updatingCompanyId();
+            CompanyManager::updatingCompanyId(playerCompanyId);
+
+            Ui::WindowManager::closeAllFloatingWindows();
+
+            CompanyManager::updatingCompanyId(previousUpdatingId);
+            setScreenFlag(ScreenFlags::networked);
+
+            // If the other party is leaving the game, go back to the title screen.
+            if (playerCompanyId != previousUpdatingId)
+            {
+                // 0x0043C1CD
+                addr<0x00F25428, uint32_t>() = 0;
+                clearScreenFlag(ScreenFlags::networked);
+                clearScreenFlag(ScreenFlags::networkHost);
+                addr<0x00508F0C, uint32_t>() = 0;
+                addr<0x00525E3C, uint8_t>() = 0;
+                addr<0x00525E3D, uint8_t>() = 0xFF;
+
+                Gfx::invalidateScreen();
+                ObjectManager::loadIndex();
+
+                Ui::WindowManager::close(Ui::WindowType::options);
+                Ui::WindowManager::close(Ui::WindowType::companyFaceSelection);
+                Ui::WindowManager::close(Ui::WindowType::objectSelection);
+
+                clearScreenFlag(ScreenFlags::editor);
+                Audio::pauseSound();
+                Audio::unpauseSound();
+
+                if (Input::hasFlag(Input::input_flags::flag5))
+                {
+                    Input::sub_407231();
+                    Input::resetFlag(Input::input_flags::flag5);
+                }
+
+                Title::start();
+
+                Ui::Windows::Error::open(StringIds::error_the_other_player_has_exited_the_game, StringIds::null);
+
+                throw GameException::Interrupt;
+            }
+        }
+
+        // 0x0043C1C8
+        exitCleanly();
+        return 0;
```

Comment:
The MSVC CI complains that this line is never reached — and it isn't, because `exitCleanly` exits the game.
```
  D:\a\OpenLoco\OpenLoco\src\OpenLoco\GameCommands\LoadSaveQuit.cpp(162): warning C4702: unreachable code [D:\a\OpenLoco\OpenLoco\src\OpenLoco\openloco.vcxproj]
```
However, if I leave it out, both gcc and clang complain…

Reply:
Just add a pragma to ignore the warning